1.5.1版本里面没有mysql8.0的依赖

![image](https://user-images.githubusercontent.com/46433683/174263504-861f674f-1162-41d2-ac42-27346ea08eb9.png)
只有mysql5.1的依赖，使用mysql8，启动报错
java.sql.SQLException: com.mysql.cj.jdbc.Driver
        at com.alibaba.druid.util.JdbcUtils.createDriver(JdbcUtils.java:679)
        at com.alibaba.druid.pool.DruidDataSource.resolveDriver(DruidDataSource.java:1214)
        at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:883)
        at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1393)
        at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1389)
        at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:100)
        at io.seata.server.storage.db.store.LogStoreDataBaseDAO.queryTableStructure(LogStoreDataBaseDAO.java:473)
        at io.seata.server.storage.db.store.LogStoreDataBaseDAO.initTransactionNameSize(LogStoreDataBaseDAO.java:457)
        at io.seata.server.storage.db.store.LogStoreDataBaseDAO.<init>(LogStoreDataBaseDAO.java:106)
        at io.seata.server.storage.db.store.DataBaseTransactionStoreManager.<init>(DataBaseTransactionStoreManager.java:94)
        at io.seata.server.storage.db.store.DataBaseTransactionStoreManager.getInstance(DataBaseTransactionStoreManager.java:79)
        at io.seata.server.storage.db.session.DataBaseSessionManager.init(DataBaseSessionManager.java:76)
        at io.seata.common.loader.EnhancedServiceLoader$InnerEnhancedServiceLoader.initInstance(EnhancedServiceLoader.java:628)
        at io.seata.common.loader.EnhancedServiceLoader$InnerEnhancedServiceLoader.createNewExtension(EnhancedServiceLoader.java:459)
        at io.seata.common.loader.EnhancedServiceLoader$InnerEnhancedServiceLoader.getExtensionInstance(EnhancedServiceLoader.java:452)
        at io.seata.common.loader.EnhancedServiceLoader$InnerEnhancedServiceLoader.loadExtension(EnhancedServiceLoader.java:420)
        at io.seata.common.loader.EnhancedServiceLoader$InnerEnhancedServiceLoader.load(EnhancedServiceLoader.java:309)
        at io.seata.common.loader.EnhancedServiceLoader$InnerEnhancedServiceLoader.access$200(EnhancedServiceLoader.java:242)
        at io.seata.common.loader.EnhancedServiceLoader.load(EnhancedServiceLoader.java:82)
        at io.seata.server.session.SessionHolder.init(SessionHolder.java:105)
        at io.seata.server.Server.start(Server.java:73)
        at io.seata.server.ServerRunner.run(ServerRunner.java:50)
        at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:808)
        at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:792)
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:346)
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:1329)
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:1318)
        at io.seata.server.ServerApplication.main(ServerApplication.java:30)
Caused by: java.lang.ClassNotFoundException: com.mysql.cj.jdbc.Driver
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        at java.lang.Class.forName0(Native Method)
        at java.lang.Class.forName(Class.java:264)
        at com.alibaba.druid.util.JdbcUtils.createDriver(JdbcUtils.java:677)
        ... 27 common frames omitted

--- a/all/pom.xml
+++ b/all/pom.xml
@@ -26,7 +26,6 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>seata-all</artifactId>
-
     <name>Seata All-in-one ${project.version}</name>
     <description>Seata is an easy-to-use, high-performance, java based, open source distributed transaction solution.</description>
 

--- a/build/pom.xml
+++ b/build/pom.xml
@@ -63,7 +63,7 @@
 
     <properties>
         <!-- seata version -->
-        <revision>1.5.2</revision>
+        <revision>1.6.0-SNAPSHOT</revision>
 
         <!-- Compiler settings properties -->
         <java.version>1.8</java.version>

--- a/build/pom.xml
+++ b/build/pom.xml
@@ -121,6 +121,8 @@
         <!-- For docker image-->
         <image.publish.skip>true</image.publish.skip>
         <image.tags>latest</image.tags>
+
+        <dependencies.copy.skip>true</dependencies.copy.skip>
     </properties>
 
     <build>

--- a/changes/en-us/1.5.2.md
+++ b/changes/en-us/1.5.2.md
@@ -1,0 +1,88 @@
+### 1.5.2
+
+[source](https://github.com/seata/seata/archive/v1.5.2.zip) |
+[binary](https://github.com/seata/seata/releases/download/v1.5.2/seata-server-1.5.2.zip)
+
+<details>	
+  <summary><mark>Release notes</mark></summary>	
+
+
+### Seata 1.5.2
+
+Seata 1.5.2 Released.
+
+Seata is an easy-to-use, high-performance, open source distributed transaction solution.
+
+The version is updated as follows:
+
+### feature：
+- [[#4661](https://github.com/seata/seata/pull/4713)] support xid consistency load balance
+- [[#4676](https://github.com/seata/seata/pull/4676)] support server to expose Nacos services by mounting SLB
+- [[#4642](https://github.com/seata/seata/pull/4642)] support batch message parallel processing
+- [[#4567](https://github.com/seata/seata/pull/4567)] support where method condition(find_in_set)
+
+
+### bugfix：
+- [[#4515](https://github.com/seata/seata/pull/4515)] fix the error of SeataTCCFenceAutoConfiguration when database unused
+- [[#4661](https://github.com/seata/seata/pull/4661)] fix sql exception with PostgreSQL in module console
+- [[#4667](https://github.com/seata/seata/pull/4682)] fix the exception in RedisTransactionStoreManager for update map During iteration
+- [[#4678](https://github.com/seata/seata/pull/4678)] fix the error of key transport.enableRmClientBatchSendRequest cache penetration if not configure
+- [[#4701](https://github.com/seata/seata/pull/4701)] fix missing command line args
+- [[#4607](https://github.com/seata/seata/pull/4607)] fix bug on skipping lock check
+- [[#4696](https://github.com/seata/seata/pull/4696)] fix oracle database insert value
+- [[#4726](https://github.com/seata/seata/pull/4726)] fix batch message send may return NullPointException
+- [[#4729](https://github.com/seata/seata/pull/4729)] fix set AspectTransactional.rollbackForClassName with wrong value
+- [[#4653](https://github.com/seata/seata/pull/4653)] fix the sql exception when pk is non-numeric in INSERT_ON_DUPLICATE SQL
+
+### optimize：
+- [[#4650](https://github.com/seata/seata/pull/4650)] fix some security vulnerabilities
+- [[#4670](https://github.com/seata/seata/pull/4670)] optimize the thread pool size of branchResultMessageExecutor
+- [[#4662](https://github.com/seata/seata/pull/4662)] optimize rollback transaction metrics
+- [[#4693](https://github.com/seata/seata/pull/4693)] optimize the console navigation bar
+- [[#4700](https://github.com/seata/seata/pull/4700)] fix maven-compiler-plugin and maven-resources-plugin execute failed
+- [[#4711](https://github.com/seata/seata/pull/4711)] separate lib dependencies for deployments
+- [[#4720](https://github.com/seata/seata/pull/4720)] optimize pom description
+- [[#4728](https://github.com/seata/seata/pull/4728)] upgrade logback dependency to 1.2.9
+- [[#4745](https://github.com/seata/seata/pull/4745)] support mysql8 in release package
+- [[#4626](https://github.com/seata/seata/pull/4626)] Replace `flatten-maven-plugin` with `easyj-maven-plugin` to fix the conflict between `shade` and `flatten`
+- [[#4629](https://github.com/seata/seata/pull/4629)] check relation of before status and after status when updating global session
+- [[#4662](https://github.com/seata/seata/pull/4662)] make EnhancedServiceLoader more readable
+
+### test:
+
+- [[#4544](https://github.com/seata/seata/pull/4544)] optimize jackson dependencies in TransactionContextFilterTest
+- [[#4731](https://github.com/seata/seata/pull/4731)] fix UT failed in AsyncWorkerTest and LockManagerTest
+
+Thanks to these contributors for their code commits. Please report an unintended omission.
+
+<!-- Please make sure your Github ID is in the list below -->
+- [slievrly](https://github.com/slievrly)
+- [pengten](https://github.com/pengten)
+- [YSF-A](https://github.com/YSF-A)
+- [tuwenlin](https://github.com/tuwenlin)
+- [Ifdevil](https://github.com/Ifdevil)
+- [wingchi-leung](https://github.com/wingchi-leung)
+- [liurong](https://github.com/robynron)
+- [opelok-z](https://github.com/opelok-z)
+- [a364176773](https://github.com/a364176773)
+- [2129zxl](https://github.com/2129zxl)
+- [Smery-lxm](https://github.com/Smery-lxm)
+- [doubleDimple](https://github.com/doubleDimple)
+- [wangliang181230](https://github.com/wangliang181230)
+- [Bughue](https://github.com/Bughue)
+- [AYue-94](https://github.com/AYue-94)
+- [lingxiao-wu](https://github.com/lingxiao-wu)
+- [caohdgege](https://github.com/caohdgege)
+
+
+Also, we receive many valuable issues, questions and advices from our community. Thanks for you all.
+
+#### Link
+
+- **Seata:** https://github.com/seata/seata
+- **Seata-Samples:** https://github.com/seata/seata-samples
+- **Release:** https://github.com/seata/seata/releases
+- **WebSite:** https://seata.io
+
+
+</details>

--- a/changes/en-us/develop.md
+++ b/changes/en-us/develop.md
@@ -4,63 +4,21 @@ Add changes here for all PR submitted to the develop branch.
 <!-- Please add the `changes` to the following location(feature/bugfix/optimize/test) based on the type of PR -->
 
 ### feature：
-  - [[#4661](https://github.com/seata/seata/pull/4713)] support xid load balance
-  - [[#4676](https://github.com/seata/seata/pull/4676)] support server to expose Nacos services by mounting SLB
-  - [[#4642](https://github.com/seata/seata/pull/4642)] support batch message parallel processing
-  - [[#4567](https://github.com/seata/seata/pull/4567)] support where method condition(find_in_set)
  
 
 ### bugfix：
-  - [[#4515](https://github.com/seata/seata/pull/4515)] fix the error of SeataTCCFenceAutoConfiguration when database unused
-  - [[#4661](https://github.com/seata/seata/pull/4661)] fix sql exception with PostgreSQL in module console
-  - [[#4667](https://github.com/seata/seata/pull/4682)] fix the exception in RedisTransactionStoreManager for update map During iteration
-  - [[#4678](https://github.com/seata/seata/pull/4678)] fix the error of key transport.enableRmClientBatchSendRequest cache penetration if not configure
-  - [[#4701](https://github.com/seata/seata/pull/4701)] fix missing command line args
-  - [[#4607](https://github.com/seata/seata/pull/4607)] fix bug on skipping lock check
-  - [[#4696](https://github.com/seata/seata/pull/4696)] fix oracle database insert value
-  - [[#4726](https://github.com/seata/seata/pull/4726)] fix batch message send may return NullPointException
-  - [[#4729](https://github.com/seata/seata/pull/4729)] fix set AspectTransactional.rollbackForClassName with wrong value
-  - [[#4653](https://github.com/seata/seata/pull/4653)] fix the sql exception when pk is non-numeric in INSERT_ON_DUPLICATE SQL
+  
 
 ### optimize：
-  - [[#4650](https://github.com/seata/seata/pull/4650)] fix some security vulnerabilities
-  - [[#4670](https://github.com/seata/seata/pull/4670)] optimize the thread pool size of branchResultMessageExecutor
-  - [[#4662](https://github.com/seata/seata/pull/4662)] optimize rollback transaction metrics
-  - [[#4693](https://github.com/seata/seata/pull/4693)] optimize the console navigation bar
-  - [[#4700](https://github.com/seata/seata/pull/4700)] fix maven-compiler-plugin and maven-resources-plugin execute failed
-  - [[#4711](https://github.com/seata/seata/pull/4711)] separate lib dependencies for deployments
-  - [[#4720](https://github.com/seata/seata/pull/4720)] optimize pom description
-  - [[#4728](https://github.com/seata/seata/pull/4728)] upgrade logback dependency to 1.2.9
-  - [[#4745](https://github.com/seata/seata/pull/4745)] support mysql8 in release package
-  - [[#4626](https://github.com/seata/seata/pull/4626)] Replace `flatten-maven-plugin` with `easyj-maven-plugin` to fix the conflict between `shade` and `flatten`
-  - [[#4629](https://github.com/seata/seata/pull/4629)] check relation of before status and after status when updating global session
-  - [[#4662](https://github.com/seata/seata/pull/4662)] make EnhancedServiceLoader more readable
+- [[#4774](https://github.com/seata/seata/pull/4774)] optimize mysql8 dependencies for seataio/seata-server image 
 
 ### test:
 
-  - [[#4544](https://github.com/seata/seata/pull/4544)] optimize jackson dependencies in TransactionContextFilterTest
-  - [[#4731](https://github.com/seata/seata/pull/4731)] fix UT failed in AsyncWorkerTest and LockManagerTest
 
 Thanks to these contributors for their code commits. Please report an unintended omission.
 
 <!-- Please make sure your Github ID is in the list below -->
 - [slievrly](https://github.com/slievrly)
-- [pengten](https://github.com/pengten)
-- [YSF-A](https://github.com/YSF-A)
-- [tuwenlin](https://github.com/tuwenlin)
-- [Ifdevil](https://github.com/Ifdevil)
-- [wingchi-leung](https://github.com/wingchi-leung)
-- [liurong](https://github.com/robynron)
-- [opelok-z](https://github.com/opelok-z)
-- [a364176773](https://github.com/a364176773)
-- [2129zxl](https://github.com/2129zxl)
-- [Smery-lxm](https://github.com/Smery-lxm)
-- [doubleDimple](https://github.com/doubleDimple)
-- [wangliang181230](https://github.com/wangliang181230)
-- [Bughue](https://github.com/Bughue)
-- [AYue-94](https://github.com/AYue-94)
-- [lingxiao-wu](https://github.com/lingxiao-wu)
-- [caohdgege](https://github.com/caohdgege)
 
 
 Also, we receive many valuable issues, questions and advices from our community. Thanks for you all.

--- a/changes/zh-cn/1.5.2.md
+++ b/changes/zh-cn/1.5.2.md
@@ -1,0 +1,89 @@
+### 1.5.2
+
+[source](https://github.com/seata/seata/archive/v1.5.2.zip) |
+[binary](https://github.com/seata/seata/releases/download/v1.5.2/seata-server-1.5.2.zip)
+
+<details>
+  <summary><mark>Release notes</mark></summary>
+
+
+### Seata 1.5.2
+
+Seata 1.5.2  发布。
+
+Seata 是一款开源的分布式事务解决方案，提供高性能和简单易用的分布式事务服务。
+
+此版本更新如下：
+
+### feature：
+- [[#4661](https://github.com/seata/seata/pull/4713)] 支持根据xid负载均衡算法
+- [[#4676](https://github.com/seata/seata/pull/4676)] 支持Nacos作为注册中心时，server通过挂载SLB暴露服务
+- [[#4642](https://github.com/seata/seata/pull/4642)] 支持client批量请求并行处理
+- [[#4567](https://github.com/seata/seata/pull/4567)] 支持where条件中find_in_set函数
+
+
+### bugfix：
+- [[#4515](https://github.com/seata/seata/pull/4515)] 修复develop分支SeataTCCFenceAutoConfiguration在客户端未使用DB时，启动抛出ClassNotFoundException的问题。
+- [[#4661](https://github.com/seata/seata/pull/4661)] 修复控制台中使用PostgreSQL出现的SQL异常
+- [[#4667](https://github.com/seata/seata/pull/4682)] 修复develop分支RedisTransactionStoreManager迭代时更新map的异常
+- [[#4678](https://github.com/seata/seata/pull/4678)] 修复属性transport.enableRmClientBatchSendRequest没有配置的情况下缓存穿透的问题
+- [[#4701](https://github.com/seata/seata/pull/4701)] 修复命令行参数丢失问题
+- [[#4607](https://github.com/seata/seata/pull/4607)] 修复跳过全局锁校验的缺陷
+- [[#4696](https://github.com/seata/seata/pull/4696)] 修复 oracle 存储模式时的插入问题
+- [[#4726](https://github.com/seata/seata/pull/4726)] 修复批量发送消息时可能的NPE问题
+- [[#4729](https://github.com/seata/seata/pull/4729)] 修复AspectTransactional.rollbackForClassName设置错误
+- [[#4653](https://github.com/seata/seata/pull/4653)] 修复 INSERT_ON_DUPLICATE 主键为非数值异常
+
+### optimize：
+- [[#4650](https://github.com/seata/seata/pull/4650)] 修复安全漏洞
+- [[#4670](https://github.com/seata/seata/pull/4670)] 优化branchResultMessageExecutor线程池的线程数
+- [[#4662](https://github.com/seata/seata/pull/4662)] 优化回滚事务监控指标
+- [[#4693](https://github.com/seata/seata/pull/4693)] 优化控制台导航栏
+- [[#4700](https://github.com/seata/seata/pull/4700)] 修复 maven-compiler-plugin 和 maven-resources-plugin 执行失败
+- [[#4711](https://github.com/seata/seata/pull/4711)] 分离部署时 lib 依赖
+- [[#4720](https://github.com/seata/seata/pull/4720)] 优化pom描述
+- [[#4728](https://github.com/seata/seata/pull/4728)] 将logback版本依赖升级至1.2.9
+- [[#4745](https://github.com/seata/seata/pull/4745)] 发行包中支持 mysql8 driver
+- [[#4626](https://github.com/seata/seata/pull/4626)] 使用 `easyj-maven-plugin` 插件代替 `flatten-maven-plugin`插件，以修复`shade` 插件与 `flatten` 插件不兼容的问题
+- [[#4629](https://github.com/seata/seata/pull/4629)] 更新globalSession状态时检查更改前后的约束关系
+- [[#4662](https://github.com/seata/seata/pull/4662)] 优化 EnhancedServiceLoader 可读性
+
+
+### test：
+- [[#4544](https://github.com/seata/seata/pull/4544)] 优化TransactionContextFilterTest中jackson包依赖问题
+- [[#4731](https://github.com/seata/seata/pull/4731)] 修复 AsyncWorkerTest 和 LockManagerTest 的单测问题。
+
+
+非常感谢以下 contributors 的代码贡献。若有无意遗漏，请报告。
+
+<!-- 请确保您的 GitHub ID 在以下列表中 -->
+- [slievrly](https://github.com/slievrly)
+- [pengten](https://github.com/pengten)
+- [YSF-A](https://github.com/YSF-A)
+- [tuwenlin](https://github.com/tuwenlin)
+- [2129zxl](https://github.com/2129zxl)
+- [Ifdevil](https://github.com/Ifdevil)
+- [wingchi-leung](https://github.com/wingchi-leung)
+- [liurong](https://github.com/robynron)
+- [opelok-z](https://github.com/opelok-z)
+- [a364176773](https://github.com/a364176773)
+- [Smery-lxm](https://github.com/Smery-lxm)
+- [lvekee](https://github.com/lvekee)
+- [doubleDimple](https://github.com/doubleDimple)
+- [wangliang181230](https://github.com/wangliang181230)
+- [Bughue](https://github.com/Bughue)
+- [AYue-94](https://github.com/AYue-94)
+- [lingxiao-wu](https://github.com/lingxiao-wu)
+- [caohdgege](https://github.com/caohdgege)
+
+同时，我们收到了社区反馈的很多有价值的issue和建议，非常感谢大家。
+
+
+#### Link
+
+- **Seata:** https://github.com/seata/seata
+- **Seata-Samples:** https://github.com/seata/seata-samples
+- **Release:** https://github.com/seata/seata/releases
+- **WebSite:** https://seata.io
+
+</details>

--- a/changes/zh-cn/develop.md
+++ b/changes/zh-cn/develop.md
@@ -3,64 +3,21 @@
 <!-- 请根据PR的类型添加 `变更记录` 到以下对应位置(feature/bugfix/optimize/test) 下 -->
 
 ### feature：
-  - [[#4661](https://github.com/seata/seata/pull/4713)] 支持根据xid负载均衡算法
-  - [[#4676](https://github.com/seata/seata/pull/4676)] 支持Nacos作为注册中心时，server通过挂载SLB暴露服务
-  - [[#4642](https://github.com/seata/seata/pull/4642)] 支持client批量请求并行处理
-  - [[#4567](https://github.com/seata/seata/pull/4567)] 支持where条件中find_in_set函数
 
 
 ### bugfix：
-  - [[#4515](https://github.com/seata/seata/pull/4515)] 修复develop分支SeataTCCFenceAutoConfiguration在客户端未使用DB时，启动抛出ClassNotFoundException的问题。
-  - [[#4661](https://github.com/seata/seata/pull/4661)] 修复控制台中使用PostgreSQL出现的SQL异常
-  - [[#4667](https://github.com/seata/seata/pull/4682)] 修复develop分支RedisTransactionStoreManager迭代时更新map的异常
-  - [[#4678](https://github.com/seata/seata/pull/4678)] 修复属性transport.enableRmClientBatchSendRequest没有配置的情况下缓存穿透的问题
-  - [[#4701](https://github.com/seata/seata/pull/4701)] 修复命令行参数丢失问题
-  - [[#4607](https://github.com/seata/seata/pull/4607)] 修复跳过全局锁校验的缺陷
-  - [[#4696](https://github.com/seata/seata/pull/4696)] 修复 oracle 存储模式时的插入问题
-  - [[#4726](https://github.com/seata/seata/pull/4726)] 修复批量发送消息时可能的NPE问题
-  - [[#4729](https://github.com/seata/seata/pull/4729)] 修复AspectTransactional.rollbackForClassName设置错误
-  - [[#4653](https://github.com/seata/seata/pull/4653)] 修复 INSERT_ON_DUPLICATE 主键为非数值异常
+
 
 ### optimize：
-  - [[#4650](https://github.com/seata/seata/pull/4650)] 修复安全漏洞
-  - [[#4670](https://github.com/seata/seata/pull/4670)] 优化branchResultMessageExecutor线程池的线程数
-  - [[#4662](https://github.com/seata/seata/pull/4662)] 优化回滚事务监控指标
-  - [[#4693](https://github.com/seata/seata/pull/4693)] 优化控制台导航栏
-  - [[#4700](https://github.com/seata/seata/pull/4700)] 修复 maven-compiler-plugin 和 maven-resources-plugin 执行失败
-  - [[#4711](https://github.com/seata/seata/pull/4711)] 分离部署时 lib 依赖
-  - [[#4720](https://github.com/seata/seata/pull/4720)] 优化pom描述
-  - [[#4728](https://github.com/seata/seata/pull/4728)] 将logback版本依赖升级至1.2.9
-  - [[#4745](https://github.com/seata/seata/pull/4745)] 发行包中支持 mysql8 driver
-  - [[#4626](https://github.com/seata/seata/pull/4626)] 使用 `easyj-maven-plugin` 插件代替 `flatten-maven-plugin`插件，以修复`shade` 插件与 `flatten` 插件不兼容的问题
-  - [[#4629](https://github.com/seata/seata/pull/4629)] 更新globalSession状态时检查更改前后的约束关系
-  - [[#4662](https://github.com/seata/seata/pull/4662)] 优化 EnhancedServiceLoader 可读性
+- [[#4774](https://github.com/seata/seata/pull/4774)] 优化 seataio/seata-server 镜像中的 mysql8 依赖
 
 
 ### test：
-  - [[#4544](https://github.com/seata/seata/pull/4544)] 优化TransactionContextFilterTest中jackson包依赖问题
-  - [[#4731](https://github.com/seata/seata/pull/4731)] 修复 AsyncWorkerTest 和 LockManagerTest 的单测问题。
 
 
 非常感谢以下 contributors 的代码贡献。若有无意遗漏，请报告。
 
 <!-- 请确保您的 GitHub ID 在以下列表中 -->
 - [slievrly](https://github.com/slievrly)
-- [pengten](https://github.com/pengten)
-- [YSF-A](https://github.com/YSF-A)
-- [tuwenlin](https://github.com/tuwenlin)
-- [2129zxl](https://github.com/2129zxl)
-- [Ifdevil](https://github.com/Ifdevil)
-- [wingchi-leung](https://github.com/wingchi-leung)
-- [liurong](https://github.com/robynron)
-- [opelok-z](https://github.com/opelok-z)
-- [a364176773](https://github.com/a364176773)
-- [Smery-lxm](https://github.com/Smery-lxm)
-- [lvekee](https://github.com/lvekee)
-- [doubleDimple](https://github.com/doubleDimple)
-- [wangliang181230](https://github.com/wangliang181230)
-- [Bughue](https://github.com/Bughue)
-- [AYue-94](https://github.com/AYue-94)
-- [lingxiao-wu](https://github.com/lingxiao-wu)
-- [caohdgege](https://github.com/caohdgege)
 
 同时，我们收到了社区反馈的很多有价值的issue和建议，非常感谢大家。

--- a/core/src/main/java/io/seata/core/protocol/Version.java
+++ b/core/src/main/java/io/seata/core/protocol/Version.java
@@ -36,7 +36,7 @@ public class Version {
     /**
      * The constant CURRENT.
      */
-    private static final String CURRENT = "1.5.2";
+    private static final String CURRENT = "1.6.0-SNAPSHOT";
     private static final String VERSION_0_7_1 = "0.7.1";
     private static final String VERSION_1_5_0 = "1.5.0";
     private static final int MAX_VERSION_DOT = 3;

--- a/pom.xml
+++ b/pom.xml
@@ -155,6 +155,9 @@
             <properties>
                 <image.publish.skip>false</image.publish.skip>
                 <image.tags>latest</image.tags>
+                <dependencies.copy.skip>false</dependencies.copy.skip>
+                <mysql.jdbc.version>5.1.35</mysql.jdbc.version>
+                <mysql8.jdbc.version>8.0.27</mysql8.jdbc.version>
             </properties>
         </profile>
         <!-- profile: arrch64 -->

--- a/server/pom.xml
+++ b/server/pom.xml
@@ -181,6 +181,55 @@
                 </configuration>
             </plugin>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>copy-dependencies</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>copy-dependencies</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${project.build.directory}/lib</outputDirectory>
+                            <excludeTransitive>false</excludeTransitive>
+                            <stripVersion>false</stripVersion>
+                            <silent>true</silent>
+                            <overWriteIfNewer>true</overWriteIfNewer>
+                            <!--resolve slf4j-simple conflicts-->
+                            <includeScope>runtime</includeScope>
+                            <excludeGroupIds>org.apache.logging.log4j,log4j,mysql</excludeGroupIds>
+                            <skip>${dependencies.copy.skip}</skip>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>copy-mysql</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>copy</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>mysql</groupId>
+                                    <artifactId>mysql-connector-java</artifactId>
+                                    <version>${mysql.jdbc.version}</version>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>mysql</groupId>
+                                    <artifactId>mysql-connector-java</artifactId>
+                                    <version>${mysql8.jdbc.version}</version>
+                                </artifactItem>
+                            </artifactItems>
+                            <outputDirectory>
+                                ${project.build.directory}/lib/jdbc
+                            </outputDirectory>
+                            <skip>${dependencies.copy.skip}</skip>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
                 <groupId>com.google.cloud.tools</groupId>
                 <artifactId>jib-maven-plugin</artifactId>
                 <version>${jib-maven-plugin.version}</version>
@@ -202,6 +251,7 @@
                         <mainClass>io.seata.server.ServerApplication</mainClass>
                         <ports>
                             <port>8091</port>
+                            <port>7091</port>
                         </ports>
                         <jvmFlags>
                             <jvmFlag>-Djava.security.egd=file:/dev/./urandom</jvmFlag>
@@ -262,6 +312,7 @@
             <properties>
                 <mysql.jdbc.version>5.1.35</mysql.jdbc.version>
                 <mysql8.jdbc.version>8.0.27</mysql8.jdbc.version>
+                <dependencies.copy.skip>false</dependencies.copy.skip>
             </properties>
             <build>
                 <finalName>seata-server</finalName>
@@ -286,53 +337,6 @@
                                 <goals>
                                     <goal>repackage</goal>
                                 </goals>
-                            </execution>
-                        </executions>
-                    </plugin>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-dependency-plugin</artifactId>
-                        <executions>
-                            <execution>
-                                <id>copy-dependencies</id>
-                                <phase>package</phase>
-                                <goals>
-                                    <goal>copy-dependencies</goal>
-                                </goals>
-                                <configuration>
-                                    <outputDirectory>${project.build.directory}/lib</outputDirectory>
-                                    <excludeTransitive>false</excludeTransitive>
-                                    <stripVersion>false</stripVersion>
-                                    <silent>true</silent>
-                                    <overWriteIfNewer>true</overWriteIfNewer>
-                                    <!--resolve slf4j-simple conflicts-->
-                                    <includeScope>runtime</includeScope>
-                                    <excludeGroupIds>org.apache.logging.log4j,log4j,mysql</excludeGroupIds>
-                                </configuration>
-                            </execution>
-                            <execution>
-                                <id>copy-mysql</id>
-                                <phase>package</phase>
-                                <goals>
-                                    <goal>copy</goal>
-                                </goals>
-                                <configuration>
-                                    <artifactItems>
-                                        <artifactItem>
-                                            <groupId>mysql</groupId>
-                                            <artifactId>mysql-connector-java</artifactId>
-                                            <version>${mysql.jdbc.version}</version>
-                                        </artifactItem>
-                                        <artifactItem>
-                                            <groupId>mysql</groupId>
-                                            <artifactId>mysql-connector-java</artifactId>
-                                            <version>${mysql8.jdbc.version}</version>
-                                        </artifactItem>
-                                    </artifactItems>
-                                    <outputDirectory>
-                                        ${project.build.directory}/lib/jdbc
-                                    </outputDirectory>
-                                </configuration>
                             </execution>
                         </executions>
                     </plugin>

